src/main/java/com/configcat/PrerequisiteFlagCondition.java

```diff
@@ -0,0 +1,37 @@
+package com.configcat;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Describes a condition that is based on a prerequisite flag.
+ */
+public class PrerequisiteFlagCondition {
+    /**
+     * The key of the prerequisite flag that the condition is based on.
```

Comment:
I'd rather move these docs comments from private fields to the public accessors.